src/components/FlawForm.vue

```diff
@@ -84,7 +84,8 @@ const errors = {
   cwe_id: null,
   major_incident_state: null,
   reported_dt: null,
-  unembargo_dt: null,
+  embargo_old_public_dt: "Set the current or a future date for an embargoed flaw.",
+  unembargo_future_public_dt: "Set the current or an old date for unembargoed flaw.",
```

Comment:
"Old date" sounds weird in this context, I would be for earlier/past/previous, wdyt?

Reply:
Right, I agree.